iped-engine/src/main/java/iped/engine/search/IPEDSearcher.java

```diff
@@ -147,7 +158,13 @@ private LuceneSearchResult searchAll() throws IOException {
         LuceneSearchResult searchResult = new LuceneSearchResult(0);
 
         // sort by index doc order: needed by features using docValues that iterate over results
-        Sort sort = new Sort(SortField.FIELD_DOC);
+        Sort sort = null;
+        if(this.sort!=null) {
+            sort = this.sort;
+        }else {
+            sort = new Sort(SortField.FIELD_DOC);
+        }
+        
```

Comment:
This will break many features that uses DocValues and iterate over search results, like filter duplicates, bookmark duplicates, find similar images or faces, metadata panel, etc. Is this really needed?

Reply:
I was needing some way to create a search to recover the other members of the track to plot them (that are siblings of a item that has a track parent). But the result should be sorted by the name, as it is the real track order, not the current selected sorting column.
I don't think this will affect other modules as they do no set the sort field, so they continue to use the value "new Sort(SortField.FIELD_DOC);"

Reply:
Ok, right, I agree. I'll add some documentation stating that the results of this search shouldn't be iterated to access DocValues, otherwise Lucene will throw an Exception.